src/client/containers/BoardPage/BoardPage.container.js

```diff
@@ -1,3 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './BoardPage.style.css';
+import OwnerBoardPage from './OwnerBoardPage';
+import MemberBoardPage from './MemberBoardPage';
 
-export const Board = () => <div className="Board ">The Board!</div>;
+export default function Board() {
+  const [boardInfo, setBoardInfo] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  // const { userId, boardId } = useParams();
+  const userId = 2;
+  const boardId = 1;
+
+  const fetchingBoardApi = async () => {
+    const API_URL = `/api/users/${userId}/boards/${boardId}`;
+    try {
+      const apiResponse = await fetch(API_URL);
+      const apiData = await apiResponse.json();
+      setBoardInfo(apiData[0]);
+      setIsLoading(false);
+    } catch (error) {
+      throw new Error(error);
+    }
+  };
+  if (isLoading) {
+    fetchingBoardApi();
+  }
+
+  return (
```

Comment:
**It is critical that you learn to identify and refactor/eliminate duplication** 
The below code can be rewritten like this

```
return (
   <div>
      {isLoading ? (
        <div>LOADING....</div>
      ) : (
        userId === boardInfo.creatorId && (
          <OwnerBoardPage boardInfo={boardInfo} />
        )
      )}
      {isLoading ? (
        <div>LOADING....</div>
      ) : (
        userId !== boardInfo.creatorId && (
          <MemberBoardPage boardInfo={boardInfo} />
        )
      )}
    </div>
  );
}
```

```
  if (isLoading) {
    return <div>LOADING....</div>
  }

  return (
    userId === boardInfo.creatorId 
      ? <OwnerBoardPage boardInfo={boardInfo}/>
      : <MemberBoardPage boardInfo={boardInfo} />
  )
  
```

Reply:
Done.